ipv8/test/dht/test_community.py

```diff
@@ -201,6 +201,12 @@ async def test_rate_limit(self):
         with self.assertRaises(TimeoutError):
             await wait_for(self.nodes[0].overlay.ping(node1), 0.1)
 
+    async def test_unload_while_contacting_node(self):
```

Comment:
[I know this PR is still WIP, but:] please assert *something* in this unit test.

Reply:
Sure, since you asked *so* nicely.